src/statement/hydrateResponse.ts

```diff
@@ -22,6 +27,13 @@ const getHydratedValue = (
     }
     return value;
   }
+  if (isByteAType(type)) {
+    if (!(value as string).startsWith(BYTE_A_PREFIX)) {
+      throw new Error("The hexadecimal string must start with \\x: " + value);
+    }
+    const valueWithoutPrefix = (value as string).substring(2);
+    return Array.from(Buffer.from(valueWithoutPrefix.toString(), "hex"));
```

Comment:
Why did you choose `Array` and not just `Buffer`?

Reply:
Because we need to return an array of bytes. Would it make more sense to just return a buffer ?

Reply:
I think you are right, found a few examples of bytea and Buffer was used - updating it :)